modules/apps/frontend-taglib/frontend-taglib-sample-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -0,0 +1,30 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+List<TabsItem> tabsItems = sampleDisplayContext.getTabsItems();
+%>
+
+<clay:tabs
```

Comment:
We should not use `<clay:tabs>` here, as they will result in frontend-only navigation. Instead, please use `<clay:navigation-bar>`.

There should be two levels of navigation, one matching taglib library, and the other matching individual tags.

Reply:
I've removed the navigation structure, since we have only one element